src/FMBot.Bot/Handlers/CommandHandler.cs

```diff
@@ -150,6 +140,18 @@ private async Task OnMessageReceivedAsync(SocketMessage s)
         }
     }
 
+    private async void TryScrobbling(SocketUserMessage msg, ICommandContext context)
+    {
+        foreach (var musicBot in MusicBot.SupportedBots)
+        {
+            if (musicBot.IsAuthor(msg.Author))
+            {
+                await this._musicBotService.Scrobble(musicBot, msg, context);
+                break;
+            }
+        }
```

Comment:
Could starting a foreach loop maybe impact performance on a large scale? Just thinking out loud, but every bot message for 200k guilds will run through this code, which can be a lot. Maybe it doesn't really matter but figured it was worth mentioning anyway, since the `MessageHandler` is also optimized to be as fast as possible.